NOTICES.txt

```diff
@@ -0,0 +1,1494 @@
+
```

Comment:
After re-reading this, I think we really need something better than this and split up into parts.

Reply:
I based this on the doc that Pivotal requires for us, and I think it's much easier to read than the typical one our org uses. What specifically would you suggest that we change?